[2023-05-11T14:51:03.198+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=tiktok_processing/run_id=manual__2023-05-11T14:50:59.340987+00:00/task_id=file_sensor permission to 509
[2023-05-11T14:51:03.303+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=tiktok_processing/run_id=manual__2023-05-11T14:50:59.340987+00:00/task_id=file_sensor permission to 509
[2023-05-11T14:51:03.434+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tiktok_processing.file_sensor manual__2023-05-11T14:50:59.340987+00:00 [queued]>
[2023-05-11T14:51:03.459+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tiktok_processing.file_sensor manual__2023-05-11T14:50:59.340987+00:00 [queued]>
[2023-05-11T14:51:03.462+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-05-11T14:51:03.514+0000] {taskinstance.py:1350} INFO - Executing <Task(FileSensor): file_sensor> on 2023-05-11 14:50:59.340987+00:00
[2023-05-11T14:51:03.522+0000] {standard_task_runner.py:57} INFO - Started process 2107 to run task
[2023-05-11T14:51:03.532+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'tiktok_processing', 'file_sensor', 'manual__2023-05-11T14:50:59.340987+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/task_7_1.py', '--cfg-path', '/tmp/tmpu6lbcqe0']
[2023-05-11T14:51:03.537+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask file_sensor
[2023-05-11T14:51:03.574+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-05-11T14:51:03.747+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=tiktok_processing/run_id=manual__2023-05-11T14:50:59.340987+00:00/task_id=file_sensor permission to 509
[2023-05-11T14:51:03.779+0000] {task_command.py:410} INFO - Running <TaskInstance: tiktok_processing.file_sensor manual__2023-05-11T14:50:59.340987+00:00 [running]> on host 694417da54ef
[2023-05-11T14:51:04.416+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tiktok_processing' AIRFLOW_CTX_TASK_ID='file_sensor' AIRFLOW_CTX_EXECUTION_DATE='2023-05-11T14:50:59.340987+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-11T14:50:59.340987+00:00'
[2023-05-11T14:51:04.464+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-05-11T14:51:04.468+0000] {filesystem.py:64} INFO - Poking for file /opt/***/data/tiktok_google_play_reviews.csv
[2023-05-11T14:51:04.483+0000] {filesystem.py:69} INFO - Found File /opt/***/data/tiktok_google_play_reviews.csv last modified: 20230504102257
[2023-05-11T14:51:04.484+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2023-05-11T14:51:04.540+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=tiktok_processing, task_id=file_sensor, execution_date=20230511T145059, start_date=20230511T145103, end_date=20230511T145104
[2023-05-11T14:51:04.644+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-11T14:51:04.774+0000] {taskinstance.py:2674} INFO - 3 downstream tasks scheduled from follow-on schedule check
